happi/cli.py

```diff
@@ -873,6 +883,125 @@ def no_create_channel(*args, **kwargs):
                     print(cache_item)
 
 
+@happi_cli.command()
+@click.pass_context
+@click.option('-f', '--file', 'ext_file',
+              type=click.Path(exists=True, dir_okay=False, resolve_path=True),
+              help='File to import additional checks from.')
+@click.option('-l', '--list', 'list_checks', is_flag=True,
+              help='List the available validation checks')
+@click.option('-c', '--check', 'check_choices', multiple=True, default=[],
+              help='Name of the check to include.  '
+                   'Can also provide a substring')
+@click.option('--glob/--regex', 'use_glob', default=True,
+              help='Use glob style (default) or regex style search terms. '
+              r'Regex requires backslashes to be escaped (eg. at\\d.\\d)')
+@click.option('--names', '-n', 'names_only', is_flag=True,
+              help='Only display names of failed entries')
+@click.argument('search_criteria', nargs=-1)
+def audit(
+    ctx,
+    list_checks: bool,
+    ext_file: Optional[str],
+    check_choices: List[str],
+    use_glob: bool,
+    names_only: bool,
+    search_criteria: Tuple[str, ...]
+):
+    """
+    Audit the current happi database.
+
+    Runs checks on the devices matching the provided SEARCH_CRITERIA.
+    Checks are simple functions that raise exceptions on failure,
+    whether naturally or via assert calls.  These functions take a single
+    happi.SearchResult as an positional argument and returns None if
+    successful.
+
+    To import additional checks, provide a file with your check function
+    and a list named ``checks`` containing the desired functions.
+    """
+    logger.debug('Starting audit block')
+    from .audit import checks, verify_result
```

Comment:
Any reason for the imports inside this function? (Here and importlib below)

Reply:
I figured that since they were only used by the audit command I didn't need to burden the rest of the file with the imports.  But I guess they're lightweight enough that we can be pythonic and move them to the top

Reply:
I see, OK. Sometimes people do this because it results in a circular import - which can (sometimes/often) be a red flag.